lib/tracer.js

```diff
@@ -0,0 +1,12 @@
+// Don't require agent until needed, early require causes its configuration to
+// be loaded from the CWD, and the CWD isn't the directory of the application
+// until run has had a chance to chdir() into it.
+
+module.exports = tracer;
+
+function tracer() {
+  // XXX sl-run.js sets the agent logger, and the quiet option, that won't
+  // happen if strongops isn't being used, unless we figure out a way to do it
+  // here.
+  return require('concurix')();
```

Comment:
Ah, I see why you made concurix a singleton. Can you not implement the singleton here?

If there is some subtleties, lets talk.